components/GothamistMainHeader.vue

```diff
@@ -1,134 +1,166 @@
+        @click="sideBarVisible = true"
+      />
+      <Sidebar
+        v-model:visible="sideBarVisible"
+        position="right"
+        data-style-mode="dark"
+        class="gothamist-sidebar px-3 md:px-4"
+      >
+        <template v-slot:header>
+          <div
+            class="gothamist-sidebar-header flex md:hidden align-items-center"
+          >
+            <v-flexible-link to="/" raw @click="menuListClick('sidebar logo')">
+              <LogoGothamist class="gothamist-sidebar-header-logo pr-2" />
             </v-flexible-link>
-            <div :class="`gothamist-header-tagline ${showLogo ? 'hidden' : 'block' } md:block`">
-                News for New Yorkers
+            <div class="gothamist-sidebar-header-tagline">
+              News for New&nbsp;Yorkers
             </div>
-        </div>
-        <div class="gothamist-header-right">
-            <v-flexible-link :to="`${donateUrl}&utm_campaign=goth_header`" raw class="pr-2">
-                <Button class="gothamist-header-donate-button p-button-rounded"> 
-                    <span class="p-button-label">Donate</span>
-                </Button>
-            </v-flexible-link>
-            <Button icon="pi pi-bars" class="p-button p-component p-button-icon-only p-button-text p-button-rounded" @click="sideBarVisible = true" />
-            <Sidebar 
-                v-model:visible="sideBarVisible" 
-                position="right" 
-                data-style-mode="dark" 
-                class="gothamist-sidebar px-3 md:px-4">
-                <template v-slot:header>
-                    <div class="gothamist-sidebar-header flex md:hidden">
-                        <v-flexible-link to="/" raw>
-                            <LogoGothamist class="gothamist-sidebar-header-logo pr-2" />
-                        </v-flexible-link>
-                        <div class="gothamist-sidebar-header-tagline">
-                            News for New Yorkers
-                        </div>
-                    </div>
-                </template>
-                <template v-slot:default>
-                    <GothamistSidebarContents
-                        :navigation="navigation" 
-                        :donateUrl="donateUrl" 
-                        class="mt-3"
-                    />
-                </template>
-            </SideBar>
-        </div>
-    </header>
+          </div>
+        </template>
+        <template v-slot:default>
+          <GothamistSidebarContents
+            :navigation="navigation"
+            :donateUrl="donateUrl"
+            class="mt-3"
+            @menuListClick="menuListClick($event)"
+          />
+        </template>
+      </Sidebar>
+    </div>
+  </header>
 </template>
 
 <style lang="scss">
 .gothamist-header {
-    width: 100%;
-    height: 68px;
-    padding: 1rem 1.5rem;
-    @include media('>md') {
-      padding: 1rem 2.5rem;
-    }
-    display: flex;
+  width: 100%;
+  height: 68px;
+  max-width: $contentWidth;
```

Comment:
added max-width and margin auto so it stays inline with the max-width of the content